src/main/java/org/takes/tk/TkProxy.java

```diff
@@ -134,7 +135,11 @@ public Response act(final Request req) throws Exception {
         com.jcabi.http.Request proxied = new JdkRequest(dest).method(method);
         final RqHeaders headers = new RqHeaders.Base(req);
         for (final String name : headers.names()) {
-            if ("content-length".equals(new EnglishLowerCase(name).string())) {
+            if ("content-length".equals(
```

Comment:
@fanifieiev please, use `Equals` here, or leave a puzzle.

Reply:
@fabriciofx [Equals](https://github.com/yegor256/cactoos/blob/master/src/main/java/org/cactoos/scalar/Equals.java) is actually is sth related to comparable, you cannot use that class for instance to check Text objects.
I think the name of the class is wrong, it should be sth like `Compared`